2006Scape Server/src/main/java/com/rs2/GameConstants.java

```diff
@@ -5,7 +5,7 @@
 	public final static boolean SERVER_DEBUG = false;
 
 	public final static String SERVER_NAME = "2006Scape", SERVER_VERSION = "Server Stage v " + GameConstants.TEST_VERSION + ".";
-	
+	public final static int SERVER_WORLD = 1;
```

Comment:
There's already a `WORLD` variable a couple lines down (on line 13) which sends the connected world to the friends list etc.

Reply:
Woops I didn't notice that😓